src/main/java/programminglife/gui/controller/GuiController.java

```diff
@@ -341,24 +344,42 @@ private void initMouse() {
 
     /**
      * Initialises the Console.
-     * @param parent is the {@link AnchorPane} in which the console is placed.
-     * @return the ConsoleView to print to
+     * @return the ConsoleView to print to.
      */
-    private ConsoleView initConsole(AnchorPane parent) {
+    private ConsoleView initConsole() {
         final ConsoleView console = new ConsoleView(Charset.forName("UTF-8"));
-        parent.getChildren().add(console);
-
-        AnchorPane.setBottomAnchor(console, 0.d);
-        AnchorPane.setTopAnchor(console, 0.d);
-        AnchorPane.setRightAnchor(console, 0.d);
-        AnchorPane.setLeftAnchor(console, 0.d);
+        AnchorPane root = new AnchorPane();
+        btnToggle.setSelected(false);
+        console.setVisible(false);
+        root.setVisible(false);
+        Stage st = new Stage();
+        st.setScene(new Scene(root, 500, 500, Color.GRAY));
```

Comment:
Please set a minimum width for the console window (for example 200x100 at least).